### DISCUSSION TITLE
Como receber notificações de novas perguntas aqui?

### DISCUSSION THREAD

**Discussion by brunosemfio** [Flutterando/forum#8] [Perguntas e Respostas] [answered]
Estou no celular a não achei uma opção para receber notificações. Apenas para se inscrever numa pergunta já feita.

**Comment by henriquearthur** [comment]
Na opção Watch do repositório, dentro da opção Custom tem o checkbox Discussions.
Não sei se tem essa opção no celular, mas pela web consegue.

**Reply by brunosemfio** [reply]
Obrigado

**Comment by ewertonls** [comment, marked as answer]
Voce pode acompanhar o repositorio ou customizar as notificaçoes:

![image](https://user-images.githubusercontent.com/79954630/189665024-b0e60c1a-6770-4bc6-b4f2-22860980e35c.png)

![image](https://user-images.githubusercontent.com/79954630/189665100-667221ad-a1c3-449f-9b69-9f356f6af41d.png)

**Reply by brunosemfio** [reply]
Obrigado
